### ISSUE TITLE
LivePersona does not show hover card

### ISSUE THREAD

**Issue by carrie-huynh** [pnp/sp-dev-fx-controls-react#1895]
#### Category
[ ] Enhancement

[x] Bug

[ ] Question

#### Version

[3.19.0]

SPFx version: 1.19.0

#### Expected / Desired Behavior / Question

LivePersona is expected to show the hover card when the control is hovered.

#### Observed Behavior
LivePersona does not show the hover card in SharePoint Online. 

However if I add the People webpart on the same page and hover over that one first, then the LivePersona works as expected. It seems like the LivePersona does not initialize/load the hover card correctly?

#### Steps to Reproduce

- Create an SPFx webpart and install **@pnp/spfx-controls-react** dependency
- Import the control into your component:

`import { LivePersona } from "@pnp/spfx-controls-react/lib/LivePersona";`

- Use the LivePersona control in your code as follows:

```
<LivePersona upn={userLogin}
  template={
    <>
      <Persona text={userDisplayName} coinSize={32} />
    </>
  }
 serviceScope={this.context.serviceScope}
/>
```

userLogin and userDisplayName are properties derived from **this.context.pageContext.user**

Either run your solution in a workbench or deploy it to your SharePoint environment. 

![LivePersona](https://github.com/user-attachments/assets/e68c0c72-4814-4ece-8a0b-6b6096d8b8cf)

**Comment by carrie-huynh** [comment]
Hi @wilecoyotegenius 

Thank you for trying to reproduce the issue I'm having with the LivePersona control. 
Interesting that it worked for you but not for me.. I wondering what went wrong with my solution.
Any ideas on how I can troubleshoot this?

**Comment by wilecoyotegenius** [comment]
Hi @carrie-huynh 

As silly as it may sound, it could be your node.js version that causes problems.

I have used node 18.20.4. Try building your web part using the same node version as I have used (remember to remove node_modules and package-lock.json before rebuilding with new node).

Moreover, I had a similar issue when using Microsoft's Live Persona in the past (PnP's Live Persona control actually uses that OOTB Microsoft's component). My web part would not work when viewed in the workbench, but started working fine, when I packaged it and dropped into an app catalog. I didn't delve on that behavior back then, but maybe this trick will solve your issue as well.

If none of the above work, please check if there are any errors in developer tools console upon hovering/clicking on the persona control - they might provide us with an useful hint for further investigation.

**Comment by carrie-huynh** [comment]
Hi @wilecoyotegenius 

I have tried the suggested actions but unfortunately it didn't solve the problem :( 

And I don't see any errors in DevTools. Here is a screenshot of the console log. Anything that looks suspicious to you?

![devtools](https://github.com/user-attachments/assets/f492901d-bb58-4e0d-8609-9b8d12ace80e)

**Comment by wilecoyotegenius** [comment]
Hi @carrie-huynh 

This is going to be a tough one. There is nothing in the console that would suggest any issues. And the persona component shows up, which suggests that loading and rendering of the internal SharePoint LivePersonaCard component has succeeded (and pnp LivePersona control has successfully completed its work). At this point, I would say Microsoft's LivePersonaCard component is to be blamed, not actual pnp's LivePersona control, which is just a wrapper.

Can you check Network tab in developer tools and verify that following files are getting loaded? Please compare names and sizes as well. Maybe your tenant is pulling some other (broken) version of livepersonacard_with-react?
![image](https://github.com/user-attachments/assets/9de1126c-3fa8-4ed3-a7de-330b885f39a2)

Finally, can you try building and running the following solution using node 18.20.4? That's the project I have used to try to reproduce the problem. Let me know if this also fails for you.  
[LivePersona.zip](https://github.com/user-attachments/files/17550002/LivePersona.zip)

**Comment by carrie-huynh** [comment]
Thanks for providing the solution. I tried running it and it worked!

Did a comparison of the our solutions and noticed that the ServiceScope defined in the LivePersona component was different. Looking back at the [documentation](https://pnp.github.io/sp-dev-fx-controls-react/controls/LivePersona/) now, I can see that I missed the part that describes that the ServiceScope should be loaded from the context of the **webpart**. Since the code sample that was provided didn't throw any errors either, I never noticed this part.

Anyhow, glad we managed to solve this. Thank you for your time!
